granulate_utils/linux/ns.py

```diff
@@ -52,6 +53,11 @@ def resolve_host_root_links(ns_path: str) -> str:
     return resolve_proc_root_links(HOST_ROOT_PREFIX, ns_path)
 
 
+def abs_path_name_parts(path: str) -> Collection[str]:
+    assert os.path.isabs(path), f"Expected {path!r} to be absolute"
+    return Path(path).parts[1:]  # skip the / (or multiple /// as .parts gives them)
```

Comment:
```suggestion
def abs_path_name_parts(path: str) -> Collection[str]:
    parts = Path(path).parts)
    assert parts[0] == "/", f"Expected {path!r} to be absolute"
    return parts[1:]
```
clearer?

Reply:
Although `isabs` deals with the case in the comment as well `//a/b` where `parts[0] == '//'` so I'll change to `startswith`